sway/commands/include_one.c

```diff
@@ -0,0 +1,78 @@
+#define _XOPEN_SOURCE 700 
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "list.h"
+#include "log.h"
+#include "unistd.h"
+#include <string.h>
+#include <libgen.h>
+#include <wordexp.h>
+
+typedef struct {
+	char *name;
+	char *path;
+} config_location_map;
+
+int compare_paths(const config_location_map* item, char* abspath) {
+	if (strcmp(item->name, basename(abspath))==0) {
+		return 0;
+	}
+	return -1;
+}
+
+void priority_configs(const char *path, const char *parent_dir, struct sway_config *config, list_t *locations) {
```

Comment:
This function works by compiling a list of includes first and populating locations?

Reply:
It was left while debugging. I will indeed remove it.

Reply:
Yes, it is compiling the list of includes and populating locations according to priority.